### ISSUE TITLE
phosphor-settingsd : preserve and sync settings

### ISSUE THREAD

**Issue by dkodihal** [openbmc/openbmc#1488]
Preserve settings across resets. This may need syncing changes in settings to a persistent storage as well. Depends on #997.

**Comment by geissonator** [comment]
Duplicate @dkodihal ?

**Comment by dkodihal** [comment]
Duplicate of #1764.
